### DISCUSSION TITLE
Rosuvastatin PBPK model

### DISCUSSION THREAD

**Discussion by NinaHanke** [Open-Systems-Pharmacology/Forum#997] [Announcement]
Dear community, 

We are happy to share our rosuvastatin model, carefully developed to model transporter-mediated DDIs. The corresponding manuscript, including comprehensive documentation, has been accepted for publication in Pharmaceutical Research [1] and the models [(Rosuvastatin, Rifampicin-Rosuvastatin-DDI, Gemfibrozil-Rosuvastatin-DDI, Probenecid-Rosuvastatin-DDI)](https://github.com/Open-Systems-Pharmacology?q=rosuvastatin&type=&language=&sort=) are uploaded as OSP repositories (tags: osp-model, ddi, pk-sim, pbpk). 

Related publication:
[1] Hanke N, Gómez-Mantilla JD, Ishiguro N, Stopfer P, Nock V. 
Physiologically Based Pharmacokinetic Modeling of Rosuvastatin to Predict Transporter-Mediated Drug-Drug Interactions. 
Pharmaceutical Research 2021. PMID: 34664206. https://doi.org/10.1007/s11095-021-03109-6

We would be very pleased, if some of you find this model helpful.

Cheers, 
Nina

**Comment by prvmalik** [comment]
Big fan, thanks, I'm giving it a try
